includes/abstracts/abstract-wc-order.php

```diff
@@ -969,7 +969,7 @@ public function apply_coupon( $raw_coupon ) {
 		if ( 0 === $this->get_customer_id() ) {
 			$data_store  = $coupon->get_data_store();
 			$usage_count = $data_store->get_usage_by_email( $coupon, $this->get_billing_email() );
-			if ( $usage_count >= $coupon->get_usage_limit_per_user() ) {
+			if ( !empty( $usage_count ) && $usage_count >= $coupon->get_usage_limit_per_user() ) {
```

Comment:
Missing one space after `!`, just to fix all coding standards.
Thanks for your help!

Reply:
@claudiosanches Thanks for the review & suggestion.